app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -189,6 +190,18 @@ class SettingsFragment : PreferenceFragmentCompat() {
             scrollToPreference(args.preferenceToScrollTo)
         }
 
+        isProfilerActive = requireContext().components.core.engine.profiler?.isProfilerActive()
```

Comment:
issue (blocking): when the profiler is started, sometimes the profiler menu item doesn't appear for me. STR:
1. Open settings, click to open secret debug settings
2. Start profiler
3. Back out to homescreen
4. Open settings

### EXpected
Profiler menu item is there

### Actual
No profiler menu item. I need to click into another settings item for it to appear. I'm not sure why this is happening.